tools/system_libs.py

```diff
@@ -1006,7 +1006,7 @@ class llvmlibc(DebugLibrary, AsanInstrumentedLibrary, MTLibrary):
   def get_files(self):
     files = glob_in_path('system/lib/llvm-libc/src/assert', '*.cpp')
     files += glob_in_path('system/lib/llvm-libc/src/complex', '**/*.cpp')
-    files += glob_in_path('system/lib/llvm-libc/src/string', '**/*.cpp')
+    files += glob_in_path('system/lib/llvm-libc/src/string', '**/*.cpp', excludes=['memset.cpp', 'memcpy.cpp'] if self.is_asan else [])
```

Comment:
The llvm libc test config is experimental, and many things are known to not work yet, including asan.

@google-yfyang is working it on it.

I'm not sure if this is the way we want to do asan integration here.. maybe?  maybe not.

Reply:
The asan tests failed on duplicate symbols, and after this change they passed. We could land this and I could run this on my CI.

Or we can say no to free coverage and not land. Whichever works.